frontend/src/components/Tabcordian.tsx

```diff
@@ -0,0 +1,107 @@
+import Link from "next/link"
+import { FC, useState } from "react"
+import { MdKeyboardArrowDown, MdKeyboardArrowRight } from 'react-icons/md'
+
+interface linkArray {
+    title: string
+    url: string
+    description: string
+}
+
+interface TabcordianData {
+    title: string
+    description: Array<string>
+    linksTitle: string
+    links: Array<linkArray>
+}
+
+export interface TabcordianProps {
+    data: Array<TabcordianData>
+}
+
+const Tabcordian: FC<TabcordianProps> = ({
+    data,
+  }: TabcordianProps) => {
+
+    const [active, setActive] = useState<number | null>(null)
```

Comment:
I'm not sure this will work if you refresh the page... you should be able to track the url and which tab is active.

Reply:
It probably wont, to be honest. though I am not entirely sure we want it to... it will work across resizes, but I don't think they have any functionality defined outside of "it's a tab and in mobile, it should accordion". So I imagine that this feature will change a lot over time.

Reply:
I looked a the mockups and I better understand the usage. LGTM